charts/ir-standalone/README.md

```diff
@@ -0,0 +1,100 @@
+# IR Standalone Deployment
+## Directory Structure
+```
+├── ir-standalone
+│   ├── Chart.yaml
+│   ├── README.md
+│   ├── etcd.yaml
+│   ├── pvc.yaml
+│   ├── templates
+│   │   ├── _helpers.tpl
+│   │   ├── configmap.yaml
+│   │   ├── frontend
+│   │   │   ├── statefulset.yaml
+│   │   │   └── svc.yaml
+│   │   ├── serviceaccount.yaml
+│   │   └── store
+│   │       ├── statefulset.yaml
+│   │       └── svc-headless.yaml
+│   └── values.yaml
+└── role_and_binding.yaml
+```
+## Prepare Dependencies (只需第一次初始化执行)
+### rbac authorization
+```
+kubectl apply -f role_and_binding.yaml
+```
+### etcd
+```
+cd GraphScope/charts
+kubectl apply -f ir-standalone/etcd.yaml
+```
+### prepare graph data
+- 在本地机器上准备图数据
+```
+# for vineyard store (resources目录下存储了modern/crew等小图的csv文件，直接拷贝到本地机器的相应目录)
+cp -r GraphScope/interactive_engine/tests/src/main/resources/ /tmp/data/
+
+# for experimental store (实验存储默认会创建modern graph, 如加载其他图需在目录graph_schema和graph_data_bin下准备相应数据)
+cp -r graph_schema /tmp/data/
```

Comment:
Can we prepare a very simple sampled data for experiment store? And refer users to `exp_store/src/bin/simple_loader` to generate the binary of certain graph.